migrations/v4.js

```diff
@@ -0,0 +1,30 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('adapt-contrib-blank - v2.0.0 > v4.1.3', async () => {
+  let blanks;
+
+  whereFromPlugin('adapt-contrib-blank - from v2.0.0', { name: 'adapt-contrib-blank', version: '<=4.1.3' });
```

Comment:
We only need to check if it's lower than the version we're updating to not equal to. Either '<4.1.3' or '<= 4.1.2'.

```suggestion
  whereFromPlugin('adapt-contrib-blank - from v2.0.0', { name: 'adapt-contrib-blank', version: '<4.1.3' });
```